src/CoinbaseSmartWallet.sol

```diff
@@ -42,6 +42,9 @@ contract CoinbaseSmartWallet is MultiOwnable, UUPSUpgradeable, Receiver, ERC1271
     /// @dev Helps enforce sequential sequencing of replayable transactions.
     uint256 public constant REPLAYABLE_NONCE_KEY = 8453;
 
+    /// @notice The address of the v0.6 EntryPoint contract.
+    address public entryPoint = 0x5FF137D4b0FDCD49DcA30c7CF57E578a026d2789;
```

Comment:
So here it's neither a constant nor immutable? Depending on the need (do we assume we only know the address at deployment or at compilation?), we should make it either a constant or an immutable.

Reply:
oops 🤦 should be immutable